docs/pages/blog/mui-x-v8.md

```diff
@@ -0,0 +1,418 @@
+  <figcaption>One of the new screens in your accounts site</figcaption>
+</figure>
+
+In addition, we've integrated telemetry into our system to continuously enhance your experience with our products.
+Using this same infrastructure, you'll soon be able to manage your license without changing your key—thanks to server-side license validation.
+
+Telemetry is released as an opt-in feature.
+We recommend enabling it by adding the following code to your project, close to your license validation:
+
+```js
+import { muiXTelemetrySettings } from '@mui/x-license';
+
+muiXTelemetrySettings.enableTelemetry();
+```
+
+You can find more details in our [telemetry guide](/x/guides/telemetry/).
+
+## Priority Support – A New Support Service
+
+A few months ago, we launched Priority Support with a few customers, and now we're excited to invite more users to take advantage of this service.
+
+Priority Support offers fast, dedicated assistance with guaranteed response times, ensuring that technical issues across MUI's full product suite are addressed swiftly and efficiently.
+
+For full details on the benefits and service levels of Priority Support, please see our [Support SLA](https://mui.com/legal/technical-support-sla/).
+
+If you're interested in upgrading to Priority Support, don't hesitate to [contact our sales team](mailto:sales@mui.com) for more information or purchase it directly through your account page.
+
+## Support for Material UI
+
+When planning this major release, our goal was to support the last two major versions of Material UI.
+However, we faced a tough choice: continue supporting older versions (v5 and v6) out of the box, or fully embrace the step forward with Material UI v7.
+With improved ESM support and seamless integration with modern bundlers like Vite and webpack, v7 represents the future of Material UI.
+Supporting both simultaneously wasn't feasible, so we chose to move forward.
+
+As a result, MUI X v8 only supports Material UI v7 by default.
+We understand this change could present a challenge, but migrating to v7 has been designed to be as smooth as possible, with comprehensive migration guides and GitHub support to assist you every step of the way.
+
+For users who want to upgrade to MUI X v8 while remaining on an earlier version of Material UI, we've prepared a [guide on usage with Material UI v5/v6](/x/migration/usage-with-material-ui-v5-v6/).
+
+## Migration guide
+
+We've published migration guides for Material UI and each of the advanced components.
+These guides detail every breaking change from previous versions with the recommended approach to fix them.
+We care deeply about providing a smooth migration, so it's been a top priority for us when planning our new major versions.
+
+Please follow the instructions in our [migration guide](/x/migration/).
+
+## Long-term support (LTS)
+
+We're now moving **MUI X v7 to long-term support**, while support for **MUI X v6 is discontinued**.
```

Comment:
This seems pretty quick. Why not have 2 years of security fixes? We barely have anything to do anyway.